lua/lush_theme/bluloco.lua

```diff
@@ -422,6 +422,55 @@ local theme = lush(function(injected_functions)
     -- BufferLineInfoSelected { fg = t.info, sp = t.primary, gui = "underline" },
     -- BufferLineHintSelected { fg = t.hint, sp = t.primary, gui = "underline" },
 
+    -- BarBar
+
+    BufferCurrent { fg = t.fg, bg = t.bg },
```

Comment:
I'd like the bufferline to be transparent also if enabled, so using Normal here would be better
```suggestion
    BufferCurrent { Normal },
```